### PR TITLE
Schema generate while polar compile

### PR DESCRIPTION
**Pull request by udit-gulati**
+ Added schema.rs to sample project.
+ schema generate command added to `polar compile`.
+ schema error handling added to Contract class.

### CODE REVIEW

File: packages/polar/sample-project/contracts/examples/schema.rs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+use std::env::current_dir;
```
Comment (coco-sha) [inline packages/polar/sample-project/contracts/examples/schema.rs:1]: what happens if user doesn't have schema in contracts ? is this case handled ?

Comment (udit-gulati) [reply packages/polar/sample-project/contracts/examples/schema.rs:1]: Yes, if schema is not there, constructor will log warnings and parseSchema will throw error

File: packages/polar/src/lib/deploy/contract.ts [hunk 2]
```diff
@@ -105,7 +105,7 @@ export class Contract {
     this.contractName = contractName.replace('-', '_');
     this.codeId = 0;
     this.contractCodeHash = "mock_hash";
-    this.contractAddress = "secret15l3z73lvy0357qrnyhcfpcxhdrxc209wpvkayl";
```
Comment (coco-sha) [inline packages/polar/src/lib/deploy/contract.ts:108]: Please use types in above functions, i am seeing few linter errors.

File: packages/polar/test/fixture-projects/compile-task-project/multiproject/contracts/contract_1/examples/schema.rs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema(&schema_for!(InitMsg), &out_dir);
+    export_schema(&schema_for!(HandleMsg), &out_dir);
+    export_schema(&schema_for!(QueryMsg), &out_dir);
+    export_schema(&schema_for!(State), &out_dir);
+    export_schema(&schema_for!(CountResponse), &out_dir);
+}
```
Comment (coco-sha) [inline packages/polar/test/fixture-projects/compile-task-project/multiproject/contracts/contract_1/examples/schema.rs:20]: why do we need this file 3 times ? could you please check if three fixture-projects aren't exactly same.
i think two of them are same.

File: packages/polar/test/fixture-projects/compile-task-project/testproject/contracts/examples/schema.rs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema(&schema_for!(InitMsg), &out_dir);
+    export_schema(&schema_for!(HandleMsg), &out_dir);
+    export_schema(&schema_for!(QueryMsg), &out_dir);
+    export_schema(&schema_for!(State), &out_dir);
+    export_schema(&schema_for!(CountResponse), &out_dir);
+}
```
Comment (coco-sha) [inline packages/polar/test/fixture-projects/compile-task-project/testproject/contracts/examples/schema.rs:20]: there are few errors in file below as well.

**Review by coco-sha — APPROVED** [review]

**Pull request merged**